javaslang-benchmark/src/test/java/javaslang/collection/VectorBenchmark.java

```diff
@@ -71,6 +71,15 @@ public void setup() {
                    && Collections.equals(clojurePersistent, javaMutable)
                    && Collections.equals(asJavaCollection(scalaPersistent), javaMutable)
                    && Collections.equals(slangPersistent, javaMutable);
+
+            Memory.storeMemoryUsages(ELEMENTS, Base.this,
```

Comment:
> How reliable is to measure memory usage in a JVM?

This measures the bytes occupied by a given object - it's not affected by the GC (if you can measure it, it's not collected yet)